README.md

```diff
@@ -1,85 +1,141 @@
-  * built in, zero-config. You give it the secret name in 1Password, it gives you the value, right in the DAG
-* Working CI, triggered via a web hook, relayed in from the outside Internet via smee.io, and secured using a 1Password secret
-  * automatically pulls from `production` when PRs are merged into the branch
-  * you can rotate the webhook token by opening 1Password, editing the entry, generating a new password, and saving it. 10 seconds, tops. 🏁
-* support for picking [environment based secrets](https://github.com/frankhereford/airflow/blob/main/dags/weather.py#L19-L23) based on local/production environment configuration
-  * zero-config in DAG, based out of `.env`
-* supports remote workers
-  * monitor their status with a [web UI](https://airflow-workers.austinmobility.io)
-    * shared credentials with admin airflow account
-* [production environment](https://airflow.austinmobility.io) which runs on a `t3a.xlarge` class instance comfortably, or `atd-data03` as well
-  * full control over [production server configuration](https://github.com/frankhereford/airflow/blob/main/airflow.cfg), yet keeping the perks of a docker stack
-* [customizable python environment](https://github.com/frankhereford/airflow/blob/main/requirements.txt) for DAGs, including [external, binary libraries](https://github.com/frankhereford/airflow/blob/main/Dockerfile#L1414-L1415) built right into the container
-  * based on bog standard `requirements.txt` & ubuntu `apt` commands
-* access to the [server's docker service](https://github.com/frankhereford/airflow/blob/main/docker-compose.yaml#L93)
-  * available on worker containers and available for DAGs
-  * you can package your ETL up as an image and then run it in the DAG 📦🐳
-    * skip installing libraries on the server
-* [flexible reverse proxy](https://github.com/frankhereford/airflow/blob/main/haproxy/haproxy.cfg#L38-L68) to distribute HTTP requests over stack
-* Interface with Airflow via Web UI, CLI and API
-  * CLI interface locally via: `docker compose run --rm airflow-cli <command>`
-* [very minimal production deployment changes](https://github.com/frankhereford/airflow/pull/34/files)
-  * server is EC2's vanilla Ubuntu LTS AMI + docker from docker's official PPA
-
-## Building multi-architecture docker images
-
-* Images created locally by default are ARM64 on a modern mac. They need to also support the server's architecture, which is AMD64.
-* This is helpful when building images to be called as the heavy-lifting component of an ETL.
-* This is not a part of building or operating the stack locally.
+# DTS Airflow stack
 
-```
-docker buildx build \
---push \
---platform linux/arm/v7,linux/arm64/v8,linux/amd64 \
---tag frankinaustin/signal-annotate:latest .
-```
+This stack is used to run DTS ETL processes and the production instance is deployed on `atd-data03`. Local development is available, and instructions are below.
```

Comment:
i'm happy to commit this change myself in a bit, but before i forget: 
```
deployed on `atd-data03` at `/usr/airflow/atd-airflow`
```

Reply:
thanks John! good call because I found myself poking around when looking for the existing Prefect and Airflow checkouts on-prem. 🎯 Just added this with another edit.